install/0000_90_cluster-version-operator_00_servicemonitor.yaml

```diff
@@ -16,3 +16,42 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-version-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    k8s-app: cluster-version-operator
+  name: cluster-version-operator
+  namespace: openshift-cluster-version
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: ClusterVersionOperatorDown
+      annotations:
+        message: ClusterVersionOperator has disappeared from Prometheus target discovery.
+      expr: |
+        absent(up{job="cluster-version-operator"} == 1)
```

Comment:
This is likely to be sufficient on its own, but we may want `namespace="openshift-cluster-version"` in the filter as well.

Reply:
https://github.com/openshift/cluster-version-operator/pull/232#discussion_r313140916